packages/flutter_tools/lib/src/commands/daemon.dart

```diff
@@ -393,6 +393,8 @@ class AppDomain extends Domain {
 
     if (options.debuggingEnabled) {
       connectionInfoCompleter = new Completer<DebugConnectionInfo>();
+      // Callback hookups' Futures are uninteresting.
+      // ignore: unawaited_futures
       connectionInfoCompleter.future.then<Null>((DebugConnectionInfo info) {
```

Comment:
`_sendAppEvent` sends the event by writing to stdout - a failure here is hopefully unlikely :)

If you want to handle it, you may want to do that by logging the failure.